casper/src/test/scala/coop/rchain/casper/batch1/MultiParentCasperCommunicationSpec.scala

```diff
@@ -26,7 +25,7 @@ class MultiParentCasperCommunicationSpec extends AnyFlatSpec with Matchers with
   "MultiParentCasper" should "ask peers for blocks it is missing" ignore effectTest {
     TestNode.networkEff(genesis, networkSize = 3).use { nodes =>
       for {
-        deploy1 <- ConstructDeploy.sourceDeployNowF(
+        deploy1 <- ConstructDeploy.sourceDeployNowF[Effect](
```

Comment:
Why not `IO` here? I've always wondered why we introduce a type for effects.

Reply:
I think this was an attempt to abstract effect type. But in this case id does not make much difference.
In principle - its a good thing to write code having abstract effect type which has to have some properties, like `Sync` (ability to suspend computation, in our case - on JVM) or `Concurrent` - ability to execute something concurrently. So then when you test you just supply effect implantation (like `IO` or `Task`) to your function and that's it.

In the case that you've mentioned - I'm not sure what is the benefit for making an alias, since here `Effect` is just an alias.